ddtrace/opentracer/tracer.go

```diff
@@ -84,3 +89,12 @@ func (t *opentracer) Extract(format interface{}, carrier interface{}) (opentraci
 		return nil, opentracing.ErrUnsupportedFormat
 	}
 }
+
+// ContextWithSpan implements opentracing.TracerContextWithSpanExtension.
+func (t *opentracer) ContextWithSpanHook(ctx context.Context, openSpan opentracing.Span) context.Context {
```

Comment:
This is code from PR https://github.com/DataDog/dd-trace-go/pull/855. Please base your changes on `v1`